clients/golang/pkg/client/proxyapi/objects.go

```diff
@@ -0,0 +1,26 @@
+package proxyapi
+
+type GetJobsResponse struct {
```

Comment:
I think maybe it is more standard to have one file per struct instead of having all in this generic file.

Reply:
ive been splitting these out into files that make sense as ive been getting more familiar with the api.